docs/Getting_Involved/Testing.rst

```diff
@@ -9,12 +9,28 @@ Executing our Tests
 *coala* has a big test suite. It is meant to work on every platform on
 every PC. If you just execute our tests you are doing us a favor.
 
-To run tests, you first need to install some dependencies. This can be
-done by executing:
+To run tests, you will need to open a file located in the default
```

Comment:
I copied this phrase from a comment given to me on a previous issue I encountered. I can change the language but I worry it won't be accepted.